internal/dataexchange/dxhttps/dxhttps.go

```diff
@@ -44,17 +44,17 @@ type HTTPS struct {
 }
 
 type wsEvent struct {
-	Type      msgType `json:"type"`
-	Sender    string  `json:"sender"`
-	Recipient string  `json:"recipient"`
-	RequestID string  `json:"requestID"`
-	Path      string  `json:"path"`
-	Message   string  `json:"message"`
-	Hash      string  `json:"hash"`
-	Size      int64   `json:"size"`
-	Error     string  `json:"error"`
-	Manifest  string  `json:"manifest"`
-	Info      string  `json:"info"`
+	Type      msgType            `json:"type"`
+	Sender    string             `json:"sender"`
+	Recipient string             `json:"recipient"`
+	RequestID string             `json:"requestId"`
+	Path      string             `json:"path"`
+	Message   string             `json:"message"`
+	Hash      string             `json:"hash"`
+	Size      int64              `json:"size"`
+	Error     string             `json:"error"`
+	Manifest  string             `json:"manifest"`
+	Info      fftypes.JSONObject `json:"info"`
```

Comment:
@peterbroadhurst @gabriel-indik I wanted to call special attention to this change. I don't actually see any dataexchange events that include "info" in their payload, so I don't think it breaks anything. Was this just added as a possible future plug point?

Reply:
@awrichar the Info field is actually expected. After reviewing the latest changes I realized this is something that is going to be coming from Data Exchange. This is, instead of having to deal with each individual type of response Data Exchange produces, the information will always be encapsulated under the "info" JSON object.